CMP1903_A1_2324/Program.cs

```diff
@@ -8,12 +8,10 @@ namespace CMP1903_A1_2324
 {
     internal class Program
     {
-        static void Main(string[] args)
+        static void Main(string[] args) //Main is used first, as all the functions of this program start from here
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+            Game DieRoller = new Game(); //creates a game object called 'DieRoller'
+            DieRoller.DieGame(); //calls the DieGame() method, which is the starting point of the program
```

Comment:
Good use of the other classes, you just make your game run by a simple line in the main program.

The program runs as it is meant to be.